packages/multimedia/libass/package.mk

```diff
@@ -3,17 +3,16 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libass"
-PKG_VERSION="0.14.0"
-PKG_SHA256="881f2382af48aead75b7a0e02e65d88c5ebd369fe46bc77d9270a94aa8fd38a2"
+PKG_VERSION="0.15.0"
+PKG_SHA256="9f09230c9a0aa68ef7aa6a9e2ab709ca957020f842e52c5b2e52b801a7d9e833"
 PKG_LICENSE="BSD"
 PKG_SITE="https://github.com/libass/libass"
 PKG_URL="https://github.com/libass/libass/releases/download/$PKG_VERSION/$PKG_NAME-$PKG_VERSION.tar.xz"
-PKG_DEPENDS_TARGET="toolchain freetype fontconfig fribidi"
+PKG_DEPENDS_TARGET="toolchain freetype fontconfig fribidi harfbuzz"
```

Comment:
I really don't like this but it seems there is no other option. libass is integral to kodi. It's probably worth moving harfbuzz out of the addon depends tree and into packages/graphics or something.

Reply:
Package moved